### ISSUE TITLE
Syntax is incorrect for alias

### ISSUE THREAD

**Issue by owenallenaz** [dupski/json-to-graphql-query#11]
We're evaluating using your package and graphql at my work, and it looks like either the package is incorrect or the documentation is incorrect, but I'm not sure which.

If I look at the alias documentation:

```js
import { jsonToGraphQLQuery } from 'json-to-graphql-query';

const query = {
    query: {
        Posts: {
            __alias: 'allPosts',
            id: true,
            comments: {
                id: true,
                comment: true
            }
        }
    }
};
const graphql_query = jsonToGraphQLQuery(query, { pretty: true });
```

In that case, I'm assuming that the Schema object is called `Posts`, and we're aliasing it to the key of `allPosts`. The intent of the `alias` mechanic in graphQL is to allow the same endpoint to be queried more than once in the same query.

In example, using the [graphIQL sandbox](https://graphql.github.io/swapi-graphql/?query=%7B%0A%20%20film1%3A%20film(id%3A%20%22ZmlsbXM6MQ%3D%3D%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20title%0A%20%20%7D%0A%20%20film2%3A%20film(id%3A%20%22ZmlsbXM6Mg%3D%3D%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20title%0A%20%20%7D%0A%7D) I can run:

```js
{
  film1: film(id: "ZmlsbXM6MQ==") {
    id
    title
  }
  film2: film(id: "ZmlsbXM6Mg==") {
    id
    title
  }
}
```

In that case I'm able to query `film` twice. The issue is with the syntax in your package, JSON does not support the same key twice in an object. IE the following is invalid JSON.

```js
{
   Posts: {
     __alias: "Test"
     ...
   }
   Posts: {
     __alias : "test2"
   }
}
```

I believe to be compatible with graphQL the keys actually needs to be flipped.

```js
{
   test: {
     __alias: "Posts"
     ...
   }
   test2: {
     __alias : "Posts"
   }
}
```

See what I'm getting at?

**Comment by dupski** [comment]
Ah yep true. I guess that's one of the reasons JSON is not used as a first class query language for GraphQL. The alias functionality was from another contributor and I'm happy to accept PRs to correct the issue.

I guess we could do something like
```
{
   test: {
     __aliasFor: "Posts"
     ...
   }
   test2: {
     __aliasFor: "Posts"
   }
}
```
To maintain compatibility with the existing syntax.

**Comment by dupski** [comment]
I'm also keen to get some other maintainers of this library on board as it is likely I won't have much time to work on it over the next few months unfortunately.

**Comment by dupski** [comment]
Resolved in `v1.7.0`
